modules/templates/helper.go

```diff
@@ -145,6 +145,9 @@ func NewFuncMap() template.FuncMap {
 		"MermaidMaxSourceCharacters": func() int {
 			return setting.MermaidMaxSourceCharacters
 		},
+		"UserLocationMapURL": func() string {
+			return setting.Service.UserLocationMapURL
+		},
```

Comment:
I do not think it's good to add more "helper" functions. You could just add it to the template data when it is needed.

Reply:
We surface other settings this way as well, and I think the idea here is that this is easier to future-proof than ensuring it's set in every view correctly?

I'm fine with either approach tbh

Reply:
Two approaches:

* for shared templates, use shared functions
* for almost every template, use context function, see my PR
    * #24056
    * #26254
    * #26385

Reply:
I agree with @wxiaoguang here, his way would be cleaner. I'm pushing a new patchset soon.